src/build/help/help.xml

```diff
@@ -2160,6 +2160,22 @@
 
                     <p>The <cmd>archive-get</cmd> command is configured and generated by <backrest/> during a restore for use by <postgres/>. See <link page="user-guide" section="/pitr">Point-in-Time Recovery</link> for an example.</p>
                 </text>
+
+                <option-list>
+                    <option id="filter" name="Filter">
+                        <summary>The absolute path to the JSON file with information about the tables for WAL filtering</summary>
+                        <text>
+                            <p>This option can be used to filter WAL for partial recovery from backup.</p>
+                            <p>The file should contain a list of tables that need to be restored. All records in WAL that do not belong to the system catalog and are not specified in the file will be replaced with NO-OP.</p>
+                            <p>If no tables are specified for the database, then only the system catalog will be restored for it. If no databases are specified, then only the following databases will be restored: template0, template1 and postgres.</p>
+                            <p>Example:</p>
+                            <p>
+                                <code>[{&quot;dbName&quot;: &quot;db1&quot;, &quot;dbOid&quot;: 20000, &quot;tables&quot;: [{&quot;tablefqn&quot;: &quot;public.t1&quot;, &quot;tableOid&quot;: 16384, &quot;tablespace&quot;: 1600, &quot;relfilenode&quot;: 16384}, {&quot;tablefqn&quot;: &quot;t2&quot;, &quot;tableOid&quot;: 16387, &quot;tablespace&quot;: 1601, &quot;relfilenode&quot;: 16385}]}, {&quot;dbName&quot;: &quot;db2&quot;, &quot;dbOid&quot;: 20001, &quot;tables&quot;: [{&quot;tablefqn&quot;: &quot;t3&quot;, &quot;tableOid&quot;: 16390, &quot;tablespace&quot;: 1700, &quot;relfilenode&quot;: 16386}]}, {&quot;dbName&quot;: &quot;db3&quot;, &quot;dbOid&quot;: 20002, &quot;tables&quot;: []}]</code>
```

Comment:
tablefqn is fully qualified name of the table (in the format <namespace>.<table_name>)